panel/models/customselect.ts

```diff
@@ -0,0 +1,67 @@
+import { Select, SelectView } from "@bokehjs/models/widgets/selectbox"
+import * as p from "@bokehjs/core/properties"
+
+
+export class CustomSelectView extends SelectView {
+    override model: CustomSelect
+
+    connect_signals(): void {
+        super.connect_signals()
+        this.connect(this.model.properties.disabled_options.change, () => this._update_disabled_options())
+      }
+
+    protected options_el(): HTMLOptionElement[] | HTMLOptGroupElement[] {
+        let opts = super.options_el()
+        opts.forEach((element) => {
+            if (this.model.disabled_options.includes(element.value)) {
+                element.setAttribute('disabled', 'true')
+            }
+        })
+        return opts
+    }
+
+    protected _update_disabled_options(): void {
+        console.log(this.input_el)
+        console.log('UPDATE DISABLED OPTIONS HERE')
+        console.log(this.model.disabled_options)
```

Comment:
Oh and please remove these.

Reply:
Fixed in https://github.com/holoviz/panel/pull/3067/commits/f713f504357aa423184ed860211e67173987f925